.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Pull Request Prerequisite Checks
+
+on:
+  pull_request:
+    branches:
+      - main
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

Comment:
doesn't make much sense using a matrix strategy if we are using only one variant. I recommend updating this to node 16 and just use that in the setup-node step

Reply:
@santam85  I removed it for now.